README.md

```diff
@@ -11,15 +11,15 @@ yarn add unionize
 ## Example
 
 Call `unionize` on a record literal mapping tag literals to value types:
-```ts
+```typescript
```

Comment:
I think 'ts' is the correct alias for typescript markup?

Reply:
both are fine AFAIK. But if there are any other changes I will revert it back. Thanks for noticing :)